test/functional/rpc_net.py

```diff
@@ -96,5 +98,34 @@ def _test_getpeerinfo(self):
         assert_equal(peer_info[0][0]['addrbind'], peer_info[1][0]['addr'])
         assert_equal(peer_info[1][0]['addrbind'], peer_info[0][0]['addr'])
 
+    def _test_getnodeaddresses(self):
+        self.nodes[0].add_p2p_connection(P2PInterface())
+        self.nodes[0].p2p.wait_for_verack()
```

Comment:
This is no longer required (`add_p2p_connection()` calls `wait_for_verack()` now)